ai21_tokenizer/jurassic_tokenizer.py

```diff
@@ -30,8 +30,11 @@ def __init__(
         self.unk_id = config.get("unk_id")
         self.eop_id = config.get("eop_id")
         self.bos_id = config.get("bos_id")
+        self.eos_id = config.get("eos_id")
 
         self._newline_piece = config.get("newline_piece")
+        self.newline_id = self.convert_tokens_to_ids(self._newline_piece)
```

Comment:
@tomeras91 I see that on line 48 we already have a `newline_id` whats the difference?

Reply:
You're right. I missed it. I'll just make it "public"

Reply:
Great. Also notice that another function `encode` is using it, so make sure you rename it in all places